The `datafile` package (https://github.com/hishamhm/datafile/) doesn't play
nicely with Nix.

We remove it and tell `luacov` where to find the asset files directly.

diff --git a/luacov-scm-1.rockspec b/luacov-scm-1.rockspec
index a8bf6d7..ecefad4 100644
--- a/luacov-scm-1.rockspec
+++ b/luacov-scm-1.rockspec
@@ -18,7 +18,6 @@ effectiveness of a test suite.
 }
 dependencies = {
    "lua >= 5.1",
-   "datafile",
 }
 build = {
    type = "builtin",
diff --git a/src/luacov/reporter/html.lua b/src/luacov/reporter/html.lua
index 26b49a3..6db32d3 100644
--- a/src/luacov/reporter/html.lua
+++ b/src/luacov/reporter/html.lua
@@ -1,4 +1,3 @@
-local datafile = require("datafile")
 local luacov_reporter = require("luacov.reporter")
 
 local reporter = {}
@@ -32,7 +31,7 @@ do
 
     -- Returns contents of a file or nil + error message.
     local function read_asset(name)
-        local f, open_err = datafile.open("src/luacov/reporter/html/static/" .. name, "rb")
+        local f, open_err = io.open("@assetDir@/src/luacov/reporter/html/static/" .. name, "rb")
 
         if not f then
             error(unprefix(open_err, name .. ": "))
